dev/io.openliberty.jakarta.concurrency.3.1_fat_tck/fat/src/io/openliberty/jakarta/enterprise/concurrent/tck/FATSuite.java

```diff
@@ -25,4 +30,39 @@
                 ConcurrentTckLauncherWeb.class //LITE MODE
 })
 public class FATSuite {
+
+    public static boolean shouldRunSignatureTests(Class<?> testClass) {
+        boolean result = false;
+        String reason = "";
+
+        try {
+            if (!testClass.isAnnotationPresent(Mode.class)) {
+                reason = testClass.getCanonicalName() + " is not run in full mode.";
+                return result = false;
+            }
+
+            if (testClass.getAnnotation(Mode.class).value() != TestMode.FULL) {
+                reason = testClass.getCanonicalName() + " is not run in full mode.";
+                return result = false;
+            }
+
+            if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
Other places I'm familiar with perform this check as follows:
System.getProperty("os.name", "unknown").toLowerCase().contains("windows")

Not sure how reliable check is with case sensitivity?  Or if "os.name" might return null on some platforms?